lib/kvdb/kvdb_rparams.c

```diff
@@ -33,67 +33,135 @@
+ *                "values": "staging"
  *            },
  *            "internal": {
- *                "keys": [ "staging" ],
- *                "values": [ "capacity" ]
+ *                "keys": "staging",
+ *                "values": "capacity"
  *            },
  *            "leaf": {
- *                "keys": [ "staging" ],
- *                "values": [ "capacity" ]
+ *                "keys": "staging",
+ *                "values": "capacity"
  *            }
  *        }
  *    },
  *    {
  *        "name": "staging_min_capacity",
  *        "config": {
  *            "root": {
- *                "keys": [ "staging" ],
- *                "values": [ "staging" ]
+ *                "keys": "staging",
+ *                "values": "staging"
  *            },
  *            "internal": {
- *                "keys": [ "capacity" ],
- *                "values": [ "capacity" ]
+ *                "keys": "capacity",
+ *                "values": "capacity"
  *            },
  *            "leaf": {
- *                "keys": [ "capacity" ],
- *                "values": [ "capacity" ]
+ *                "keys": "capacity",
+ *                "values": "capacity"
+ *            }
+ *        }
+ *    },
+ *    {
+ *        "name": "pmem_only",
+ *        "config": {
+ *            "root": {
+ *                "keys": "pmem",
+ *                "values": "pmem"
+ *            },
+ *            "internal": {
+ *                "keys": "pmem",
+ *                "values": "pmem"
+ *            },
+ *            "leaf": {
+ *                "keys": "pmem",
+ *                "values": "pmem"
+ *            }
+ *        }
+ *    },
+ *    {
+ *        "name": "pmem_staging",
+ *        "config": {
+ *            "root": {
+ *                "keys": "pmem",
+ *                "values": "pmem"
+ *            },
+ *            "internal": {
+ *                "keys": "pmem",
+ *                "values": "staging"
+ *            },
+ *            "leaf": {
+ *                "keys": "pmem",
+ *                "values": "staging"
+ *            }
+ *        }
+ *    },
+ *    {
+ *        "name": "pmem_capacity",
+ *        "config": {
+ *            "root": {
+ *                "keys": "pmem",
+ *                "values": "pmem"
+ *            },
+ *            "internal": {
+ *                "keys": "pmem",
+ *                "values": "capacity"
+ *            },
+ *            "leaf": {
+ *                "keys": "pmem",
+ *                "values": "capacity"
+ *            }
+ *        }
+ *    },
+ *    {
+ *        "name": "pmem_staging_capacity",
+ *        "config": {
+ *            "root": {
+ *                "keys": "pmem",
+ *                "values": "pmem"
+ *            },
+ *            "internal": {
+ *                "keys": "staging",
+ *                "values": "staging"
+ *            },
+ *            "leaf": {
+ *                "keys": "staging",
+ *                "values": "capacity"
  *            }
  *        }
  *    }
```

Comment:
Who wants to get rid of this comment?  The setup code in `mclass_policies_default_builder` is easier to read now that mblock allocation fallback has been removed.

Reply:
Agreed, I'm getting rid of it, don't see a value...

Reply:
Comment exists because it previously existed when I ported code over. The only reason I see it as useful is that there aren't many example of mclass_policies being set.

Reply:
We don't support custom mclass policy anyways, so not that useful IMO.

Reply:
Feels like we should just remove the rparam then. What is the point of exposing something we don't even support?

Reply:
Sorry, I should have said we don't support it now but have all the infrastructure to do if needed in future.. It's an experimental rparam.